packages/liveblocks-node/src/client.ts

```diff
@@ -1346,6 +1346,21 @@ export class Liveblocks {
         : undefined,
     };
   }
+
+  public async triggerInboxNotification(params: {
+    userId: string;
+    kind: `$${string}`;
+    roomId?: string;
+    subjectId: string;
+    activityData: BaseMetadata;
```

Comment:
They're the same types but for clarity we should use `ActivityData` from `InboxNotificationData.ts` here.

Reply:
Fixed!